dependency/vault_write_test.go

```diff
@@ -71,6 +71,69 @@ func TestNewVaultWriteQuery(t *testing.T) {
 	}
 }
 
+func TestVaultWriteSecretKV_Fetch(t *testing.T) {
+	t.Parallel()
+
+	// trigger a nil-pointer-deref panic in wq.Fetch() with KVv1 due to
```

Comment:
these comments may seem cryptic to people looking at this test after the fix is in

Reply:
Is there some way I could make the clearer? There are times when I'm looking at a test and wondering why it was added... so I thought I'd start putting in small comments to see if that helped.

Reply:
Yeah that would be useful. You could add a reference to the GH issue for context.

Something like:
```
// previously triggered a nil-pointer-deref panic in wq.Fetch() with KVv1 due to
// writeSecret() returning nil for vaultSecret
// see #1255
```

Reply:
Thanks. I'll use GH-1252 to make it clear it is a github issue. 

I'll make a similar change (past tense + issue reference) to the other test's comment as well.

Thanks!